.github/CODEOWNERS

```
@@ -263,7 +263,7 @@
 /sdk/containerregistry/Microsoft.*/ @toddysm @yugangw-MSFT
 
 # ServiceLabel: %Cosmos %Service Attention
-/sdk/cosmosdb/          @Wmengmsft @MehaKaushik @shurd @anfeldma-ms @zfoster @kushagraThapar @southpolesteve
+/sdk/cosmosdb/          @pjohari-ms @MehaKaushik @shurd @anfeldma-ms @zfoster @kushagraThapar @southpolesteve
```

```suggestion
/sdk/cosmosdb/          @pjohari-ms @MehaKaushik @shurd @anfeldma-ms @zfoster @kushagraThapar
```

Steve doesn't work here anymore.   The rest of this list doesn't match the bot rules...  we should figure out if that's intended.   If we ever do regenerate via tool, all of these folks will end up there.